source/linux/util.js

```diff
@@ -4,6 +4,25 @@ import {promises as fsPromises} from 'node:fs';
 
 export const execFile = promisify(childProcess.execFile);
 export const exec = promisify(childProcess.exec);
+export const spawn = (
+	cmd,
+	args,
+) => new Promise(resolve => {
+	const cp = childProcess.spawn(cmd, args);
+	cp.on('spawn', () => {
```

Comment:
This needs a code comment on why we resolve on `spawn`, which makes no sense without the context.